scripts/enum/trait.lua

```diff
@@ -0,0 +1,136 @@
+xi = xi or {}
+
+xi.trait =
+{
+    -- IF YOU ADD ANY NEW ENTRY HERE, ADD IT IN src/map/trait.h ASWELL!
+
```

Comment:
request: add comment from the doc
> Offset by 1536 from ROM\181\72.dat

in both lua and the .h file

and then nuke doc coz it'll be redundant

Reply:
Rebased, all comments adressed